packages/cheqd/src/tokenStatusList/CheqdTokenStatusListService.ts

```diff
@@ -0,0 +1,159 @@
+import { AgentContext, CredoError, getKeyFromVerificationMethod, Buffer, utils } from '@credo-ts/core'
+import { StatusListPayload, StatusListToken } from './types'
+import { createEmptyBitmap, decodeBitmap, encodeBitmap, isBitSet, setBit } from './utils/bitmap'
+import { CheqdCreateResourceOptions, CheqdDidResolver } from '../dids'
+import { TokenStatusList } from './types/tokenStatusList'
+import { CheqdApi } from '../CheqdApi'
+import { parseCheqdDid } from '../anoncreds/utils/identifiers'
+import base64url from 'base64url'
+
+export class CheqdTokenStatusListService implements TokenStatusList {
+  private async loadStatusList(
+    agentContext: AgentContext,
+    statusListId: string
+  ): Promise<{
+    metadata: any
+    bitmap: Buffer
+    jwt: string
+  }> {
+    const api = agentContext.dependencyManager.resolve(CheqdApi)
+    const resource = await api.resolveResource(statusListId)
+    const jwt = resource.resource?.data.toString()
+    const payload = JSON.parse(Buffer.from(jwt!.split('.')[1], 'base64').toString()) as StatusListPayload
```

Comment:
WE have tools like the `Jwt.fromSerializedJwt` to parse JWTs. 

Also should we not verify it first?